manifests/local/deployment.yaml

```diff
@@ -9,6 +9,8 @@ groups:
     path: manifests/local/networking-modules.yaml
   - name: storage
     path: manifests/local/storage-modules.yaml
+  - name: replication
+    path: manifests/local/replicator-modules.yaml
```

Comment:
Is replication now required for EKS?  If this is a local deployment (as opposed to local-isolated), what is the need for the image replication?